timeless/views.py

```diff
@@ -13,17 +14,52 @@ class CommentView(CrudView):
          model = Comment
          url_lookup = "comment_id"
 
-    @todo #123:30min Continue with the implementation of CrudAPIView.
-     Implement post, put and delete methods. We should return json
-     representation of object model in methods.
+    @todo #221:30min Remove references to database objects. we should not tie
+     our view layer to our database objects. We should create some form of
+     abstraction to handle the model calls, for example,
+     model.query.get(object_id) that would decorate the real implementation,
+     eliminating the coupling here.
     """
     model = None
     url_lookup = None
 
-    def get(self, *args, **kwargs):
```

Comment:
@paulodamaso oops, this method is correct and should be here. but without *args, **kwargs

Reply:
@vryazanov I don't think that this implementation is correct: it makes reference to `kwargs` and does not return an json with the object data, so I've decided to remove it

Reply:
@paulodamaso be honestly, I’m confused. Mb I was wrong all time about this and there is a way to pass url args to methods.. %) anyway, lets leave it as it is now

Reply:
@paulodamaso I looked at the code of router and method views and realized that we actually can pass url args to methods (pass id to `get`, `delete`, `put`), just need to register views in router by specific way. It means I wasted your time in a few cases. It's my fault, sorry about that.

Reply:
@vryazanov Don't worry, I should have proven you that it is possible. Let's just leave it as is for now, we will use the parameters later if we need them